Makefile

```diff
@@ -0,0 +1,26 @@
+generator=typescript-node
+openapi-generator-cli=docker run --rm -v ${PWD}:/local -w /local openapitools/openapi-generator-cli:v5.4.0
```

Comment:
Why use a makefile in favor of the Github action for openApi generator cli?

Reply:
I'm using Docker because I need to run the generator locally and the same version/image will be used in CI. The action can only run from Github servers in a non-interactive way. The Makefile file is just a generic build tool to trigger such commands easily (e.g. `make models`).